sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/src/main/java/com/microsoft/opentelemetry/exporter/azuremonitor/AzureMonitorExporterBuilder.java

```diff
@@ -143,22 +143,23 @@ public AzureMonitorExporterBuilder configuration(Configuration configuration) {
      * @throws IllegalArgumentException If the connection string is invalid.
      */
     public AzureMonitorExporterBuilder connectionString(String connectionString) {
-        this.instrumentationKey = extractInstrumentationKey(connectionString);
+        this.instrumentationKey = extractValueFromConnectionString(connectionString, "InstrumentationKey");
```

Comment:
maybe extract once into Map<String, String> once and pull values from there? not a big deal tho

Reply:
Done